source/guide_kirby.rst

```diff
@@ -0,0 +1,161 @@
+.. sectionauthor:: Daniel Kratz <uberlab@danielkratz.com>
+.. highlight:: console
+
+.. sidebar:: Logo
+  
+  .. image:: _static/images/kirby.svg 
+      :align: center
+
+#########
+Kirby
+#########
+
+Kirby_ is a modern, fast, flexible, file-based (no database) content management system written in PHP. 
+
+The idea behind Kirby is to provide a tool for developing websites or applications that is both easy to use for developers and at the same time gives you all the freedom you want to structure your content and render your output.
+
+Kirby was released for the first time in 2009 and is actively developed by the Bastian Allgeier GmbH. Although the code is open source, a license is required for use in production. 
```

Comment:
open source is defined as 
> software for which the original source code is made freely available and may be redistributed and modified.

This is not the case for kirby as [the license](https://github.com/getkirby/kirby/blob/master/license.md) states the following
> You may not (...) Distribute derivative works (...); (...) distribute, or otherwise transfer or make available the Software or its Source Code, in whole or in part, in any form to any third parties.

Please change this to "Although the code of kirby is available for download, ...". Thanks!